### DISCUSSION TITLE
BelongsToMany not working in v1.2.6 anymore

### DISCUSSION THREAD

**Discussion by Geraldhub** [wintercms/winter#1122] [General]
The following model definition:

    public $belongsToMany = [
        'items' => [
            'abuseio\scart\models\Input',
            'table' => 'abuseio_scart_input_parent',
            'key' => 'parent_id',
            'parentKey' => 'id',
            'otherKey' => 'input_id',
            'relatedKey' => 'id',
            'conditions' => 'abuseio_scart_input_parent.deleted_at is null',
            ],
    ];

Sincs v1.2.6 this gives the error:

TypeError: Winter\Storm\Database\Relations\BelongsToMany::shouldSelect(): Argument #1 ($columns) must be of type array, int given, called in /mnt/AbuseIO/EOKM/SCART/source/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Relations/BelongsToMany.php on line 922 and defined in /mnt/AbuseIO/EOKM/SCART/source/vendor/winter/storm/src/Database/Relations/Concerns/BelongsOrMorphsToMany.php:59

It looks like something is broken? 

Rollback to v1.2.5 any this is works again.

**Comment by mjauvin** [comment]
Duplicate of https://github.com/wintercms/winter/issues/1110

**Reply by Geraldhub** [reply]
Thanks! 

Overlook 1110 because of a search on "BelongToMany" only
